src/NpmUtilities.js

```diff
@@ -30,6 +30,25 @@ export default class NpmUtilities {
   }
 
   @logger.logifySync()
+  static getWhoIAm() {
+    try {
+      return ChildProcessUtilities.execSync("npm whoami", { stdio: "none" });
+    } catch (e) {
+      return undefined;
+    }
+  }
+
+  @logger.logifySync()
+  static getConfig(key) {
+    return ChildProcessUtilities.execSync(`npm config get ${key}`);
+  }
+
+  @logger.logifySync()
+  static setConfig(key, value) {
```

Comment:
Is `setConfig()` meant to be used anywhere, or is this added just to make the API symmetrical?